modules/ha/pages/elastic-cluster.adoc

```diff
@@ -0,0 +1,43 @@
+= Compute-on-demand Elastic Cluster
+:description: Overview of TigerGraph's compute-on-demand ER cluster.
+
+TigerGraph allows you to spin up an Elastic Read-only (ER) cluster from your primary cluster to handle compute-intensive online analytical processing (OLAP) queries on demand.
+Operations on the ER cluster do not interfere with the primary cluster in any way.
+
+NOTE: This feature is currently in the preview stage.
+If you want to set up elastic clusters for your production environment, contact mailto:support@tigergraph.com[].
+
+The elasticity afforded by on-demand ER clusters can provide the following benefits:
+
+* Better availability.
+Diverting memory-intensive and time-consuming OLAP queries to an isolated ER cluster allows your primary cluster to have better availability for other workloads.
+* Better cost management.
+Provisioning dedicated machines to process OLAP can prove quite expensive.
+When the queries are not running, the excess hardware sits idle, which is a waste of resources.
+The elasticity of ER clusters make sure that you are only paying for computing power you need when you need it.
+
+
+== Architecture
+The ER cluster shares storage with the primary cluster.
+When the ER cluster is set up, it reads from the shared storage to retrieve all data from the primary cluster.
+After setup, it no longer receives updates from the primary cluster.
+
+.Architectural diagram for ER cluster
+image:elastic-cluster.png[]
+
+The ER clusters are able to process memory-intensive OLAP queries in an isolated environment with no impact to the primary cluster.
+After the queries finish, the ER cluster should be shut down.
+The next time when there is a demand for a memory-intensive OLAP query, you can set up a new ER cluster to meet the demand.
+
+== Use cases
+If your workload involves online analytical processing (OLAP) queries that access large amounts of data and are very time-consuming, you can benefit from using ER clusters for your OLAP queries.
```

Comment:
Another huge impact scenario is to allow running OLAP queries while primary cluster is taking a backup.